tests/modules/suite2p_wrapper/resources/default_s2p_args.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
Why do we need to store these in our repo? Aren't these [here](https://github.com/MouseLand/suite2p/blob/v0.10.3/suite2p/ops/ops.npy) ?

Reply:
Yes, but our default environment doesn't actually include Suite2P. Suite2P tests are only run in our suite2p-enabled Docker image (now that we know we are using Suite2P for motion correction and segmentation, we can probably revisit the wisdom of keeping two Docker images).

I wanted to be able to run this test without having Suite2P installed.

Reply:
Ah, in that case could you please move this to the test folder?

Reply:
Nevermind :)